CleverPushLocation/Source/CleverPushLocation.m

```diff
@@ -7,6 +7,13 @@
 @implementation CleverPushLocation
 
 CLLocationManager* locationManager;
+NSTimer *geoFenceTimer;
+double geoFenceTimerDelay;
+bool geoFenceTimeoutCompleted;
+NSMutableArray *geoFenceArray;
```

Comment:
Find better name for this variable. geoFenceArray is misleading because it only contains geoFences which have delays. Maybe "delayedGeoFences"